agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java

```diff
@@ -355,7 +355,10 @@ private DiseaseDownloadRow getBaseDiseaseDownloadRow(DiseaseAnnotationDocument a
 			for (org.alliancegenome.curation_api.model.entities.DiseaseAnnotation da : annotation.getPrimaryAnnotations()) {
 				if (CollectionUtils.isNotEmpty(da.getConditionRelations())) {
 					List<ConditionRelation> conditionModifiers = da.getConditionRelations().stream().filter(conditionRelation -> conditionRelation.getConditionRelationType() != null)
-						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition") || conditionRelation.getConditionRelationType().getName().contains("induced")).toList();
+						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition") 
+						|| conditionRelation.getConditionRelationType().getName().contains("induced") 
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace
<!-- __reviewdog__:ChBmYzhlMWQ5MDc1MjJlOGEwEgpjaGVja3N0eWxl -->